sync.yml

```diff
@@ -0,0 +1,16 @@
+group:
```

Comment:
Since this configuration is related to the github action i would suggest to move this file into to : `.github/config/sync-config.yml`

And add a comment at the beginning of the file: `configuration for the github action : BetaHuhn/repo-file-sync-action`